Makefile

```diff
@@ -889,6 +889,8 @@ update-js: node-check | node_modules
 	npx updates -u -f package.json
 	rm -rf node_modules package-lock.json
 	npm install --package-lock
+	npx nolyfill install
+	npm update
```

Comment:
Don't think we need this here. Dependencies were already updated in `npx updates` above. We only want to refresh the "nolyfills".